pkg/block/metrics.go

```diff
@@ -6,9 +6,19 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/treeverse/lakefs/pkg/httputil"
 )
 
+var concurrentOperations = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "blockstore_concurrent_operations",
+		Help: "Number of concurrent blockstore operations",
+	},
+	[]string{"operation", "blockstore_type"},
```

Comment:
Consider:
```suggestion
	[]string{"blockstore_type", "operation"},
```
which more closely matches other metrics.

If you make this change, please be sure to fix all the "WithLabelValues" calls, of course.

Reply:
Looking at different metrics we have with the operation field, it seem it is consistently the first field, so I would prefer to keep it as is.